src/evict/evict_lru.c

```diff
@@ -303,79 +303,81 @@ __wt_evict_thread_run(WT_SESSION_IMPL *session, WT_THREAD *thread)
 		__wt_epoch(session, &cache->stuck_ts);
```

Comment:
Yes, good catch and a bad side effect.  My choices are to move it, or add an initialization function for threads in a group.  I'll move it.